kernel/src/process_policies.rs

```diff
@@ -4,9 +4,9 @@
 
 //! Process-related policies in the Tock kernel.
 //!
-//! This file contains definitions and implementations of policies the Tock
-//! kernel can use when managing processes. For example, these policies control
-//! decisions such as whether a specific process should be restarted.
+//! This file contains definitions of policies the Tock kernel can use when
+//! managing processes. For example, these policies control decisions such as
+//! whether a specific process should be restarted.
```

Comment:
This file / description is a bit weird now since it doesn't actually contain any policies anymore?

Reply:
It still defines what we need policies for.